dependencies/supported_version/Trilinos.txt

```diff
@@ -1 +1 @@
-1eab15637f2998d1e86fe127b78200f2c9687cb5 2025.1 none
\ No newline at end of file
+11a16481fa1af48e3cfdb0aa84c29617ce94479f 2025.1 16.1.0
```

Comment:
@maxfirmbach Please change this as follows:

```suggestion
11a16481fa1af48e3cfdb0aa84c29617ce94479f 2025.2 16.1.0
1eab15637f2998d1e86fe127b78200f2c9687cb5 2025.1 none
```

This way, the new version works, as well as the old version. Otherwise everybody will get warnings when running with the old version.
Sorry if that was not clear. Maybe we should document it in the CMake/dependency internal docs?

Reply:
@sebproell What is the plan for testing the supported versions? If the new dependencies image contains the new Trilinos version, we don't know if the old version still works.

Reply:
@maxfirmbach we should not drop support for an old version out of nowhere. With your proposed change, everybody using the old version will get a warning that the version is unknown. 4C then assumes that their version is _more recent_ than the latest known version, which is wrong. Yes, users may fix this by specifying the version manually, but this is not really intended here. I specifically designed the mechanism for what you are trying to do here: if you add a new version, you add it. Removing an old version is breaking. I think we should only do this in releases. 
Maybe we even need to deprecate it first?